File: app/src/main/java/gr/tsagi/jekyllforandroid/app/util/NavDrawerItem.java
package gr.tsagi.jekyllforandroid.app.util;

/**
 * Created by tsagi on 7/9/14.
 */
public class NavDrawerItem {

    private String title;
    private int icon;


    public NavDrawerItem(){}

    public NavDrawerItem(String title, int icon){
        this.title = title;
        this.icon = icon;
    }

    public String getTitle(){
        return this.title;
    }

    public int getIcon(){
        return this.icon;
    }

    public void setTitle(String title){
        this.title = title;
    }

    public void setIcon(int icon){
        this.icon = icon;
    }

}

File: app/src/main/java/gr/tsagi/jekyllforandroid/app/utils/NavDrawerItem.kt
package gr.tsagi.jekyllforandroid.app.utils

/**
 * Created by tsagi on 7/9/14.
 */
class NavDrawerItem {

    var title: String? = null
    var icon: Int = 0


    constructor() {}

    constructor(title: String, icon: Int) {
        this.title = title
        this.icon = icon
    }

}